yt_dlp/extractor/gedidigital.py

```diff
@@ -1,123 +1,279 @@
+            'duration': 122,
+            'timestamp': 1670866318,
+            'upload_date': '20221212',
+            'formats': 'count:10',
+        },
+        'params': {
+            'skip_download': True,
+        },
     }, {
-        'url': 'https://video.lanuovaferrara.gelocal.it/locale/due-bombole-di-gpl-aperte-e-abbandonate-i-vigili-bruciano-il-gas/134391/134957',
-        'only_matching': True,
+        # only audio
+        'url': 'https://ilpiccolo.gelocal.it/onepodcast/deejay/2022/12/08/audio/episodio_6_lingegnere-10858264/',
+        'info_dict': {
+            'id': 'deejay:podcast:348159',
+            'ext': 'm4a',
+            'title': 'Episodio 6: L’ingegnere',
+            'description': 'md5:ba4a655f864fb998f3fdba807ed85a66',
+            'thumbnail': r're:^https://.+?800x800\.jpg',
+            'duration': 3571,
+            'timestamp': 1670472004,
+            'upload_date': '20221208',
+            'formats': 'count:3',
+        },
+        'params': {
+            'skip_download': True,
+        },
     }, {
-        'url': 'https://video.corrierealpi.gelocal.it/dossier/cortina-2021-i-mondiali-di-sci-alpino/mondiali-di-sci-il-timelapse-sulla-splendida-olympia/133760/134331',
-        'only_matching': True,
+        # audio playlist (podcast and alike)
+        'url': 'https://www.lastampa.it/rubriche/daytime/2022/05/09/playlist/daytime-3447164',
+        'info_dict': {
+            'id': '3447164',
+            'title': 'DayTime',
+        },
+        'playlist_count_min': 100,
     }, {
-        'url': 'https://video.lasentinella.gelocal.it/locale/vestigne-centra-un-auto-e-si-ribalta/138931/139466',
-        'only_matching': True,
+        # playlist with single episode selection
+        'url': 'https://www.lastampa.it/rubriche/daytime/2022/05/09/playlist/daytime-3447164#gnn:audio:5427139',
+        'info_dict': {
+            'id': 'gnn:audio:5427139',
+            'ext': 'm4a',
+            'title': 'Una legge di civiltà',
+            'description': 'md5:963f7397192d33f05b321f00f4a60e30',
+            'thumbnail': r're:^https://.+?thumb-full.+?\.jpg',
+            'duration': 322,
+            'timestamp': 1656374400,
+            'upload_date': '20220628',
+            'formats': 'count:3',
+        },
+        'params': {
+            'skip_download': True,
+        },
     }, {
-        'url': 'https://video.espresso.repubblica.it/tutti-i-video/01-ted-villa/14772',
+        # new url for video in GediDigitalLegacy
+        'url': 'https://espresso.repubblica.it/video/2020/10/08/video/festival_emergency_villa_la_buona_informazione_aiuta_la_salute_-321887259/',
         'only_matching': True,
     }]
+    _WEBPAGE_TESTS = [{
+        'url': 'https://nixxo.github.io/yt-dpl-test-pages/gedidigital-iframes.html',
+        'info_dict': {
+            **_TESTS[1]['info_dict'],
+            # description in embeds is cropped from original lenght
+            'description': 'md5:054880e1a8463570348d1c104a0ca690'
```

Comment:
pls explain

Reply:
description in the normal webpage is full length, in the embed page is cropped (in the page itself, not by the extractor).

So md5 is different.

Reply:
Can we transform the embed URL to normal URL before `_download_webpage`? If so, this won't be an issue

Reply:
didn't thought of that. Done.